examples/mpi_helloworld.cpp

```diff
@@ -0,0 +1,58 @@
+#include <faabric/util/logging.h>
+
+#include <faabric/mpi-native/MpiExecutor.h>
+#include <faabric/mpi/mpi.h>
+
+#include <unistd.h>
+
+int main(int argc, char** argv)
```

Comment:
This main function is what we'd need to add to standalone MPI binaries, and then provide a definition for `int faabric::executor mpiFunc()`.